lib/common/socket/evloop.c.h

```diff
@@ -438,7 +438,8 @@ h2o_socket_t *h2o_evloop_socket_accept(h2o_socket_t *_listener)
 
     /* cache the remote address, if we know that we are going to use the value (in h2o_socket_ebpf_lookup_flags) */
 #if H2O_USE_EBPF_MAP
-    struct sockaddr_storage peeraddr[1];
+    struct sockaddr_storage _peeraddr;
+    struct sockaddr_storage *peeraddr = &_peeraddr;
```

Comment:
As talked offline, I rather focused that the types of the variable differs depending on the build flag, and I think it's not polite since `sizeof(peeraddr)` gives us a big surprise. Suppressing the warning is the original motivation of this PR but it's just a bonus now. However this is not a big matter, it's your choice 🙂

Reply:
Thanks, I think I can accept that argument (that types should be equivalent on both sides). Maybe something like above?